gulpfile.js

```diff
@@ -160,6 +160,12 @@ const parseColor = color => {
   };
 };
 
+const parseToken = token => {
```

Comment:
Can we add a comment explaining what this function does? It's not entirely clear from the code

Reply:
I can go one better
<img width="218" alt="image" src="https://user-images.githubusercontent.com/30267516/78047339-21dec380-7370-11ea-9227-d2686b2c232f.png">